keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -170,7 +169,7 @@ Since images can be pre-loaded, loaded outside the `kubelet` process, and
 garbage collected.. the list of images that required authentication in `kubelet`
```

Comment:
> ### Notes/Constraints/Caveats (Optional)
>
> With the default of the feature gate being off, users / cloud providers will have
to set the feature gate to true to gain these this Secure by Default benefit.

Will this change when the feature is GA?

Reply:
yes, by GA the gate will be removed

Reply:
I would clarify this in the KEP.